pkg/cmd/project/item-edit/item_edit.go

```diff
@@ -0,0 +1,286 @@
+			return printItemJSON(config, &query.Update.Item)
+		}
+
+		return printItemResults(config, &query.Update.Item)
+	}
+
+	config.tp.AddField("No changes to make")
+	return config.tp.Render()
+
+}
+
+func buildEditDraftIssue(config editItemConfig) (*EditProjectDraftIssue, map[string]interface{}) {
+	return &EditProjectDraftIssue{}, map[string]interface{}{
+		"input": githubv4.UpdateProjectV2DraftIssueInput{
+			Body:         githubv4.NewString(githubv4.String(config.opts.body)),
+			DraftIssueID: githubv4.ID(config.opts.itemID),
+			Title:        githubv4.NewString(githubv4.String(config.opts.title)),
+		},
+	}
+}
+
+func buildUpdateItem(config editItemConfig, date time.Time) (*UpdateProjectV2FieldValue, map[string]interface{}) {
+	var value githubv4.ProjectV2FieldValue
+	if config.opts.text != "" {
+		value = githubv4.ProjectV2FieldValue{
+			Text: githubv4.NewString(githubv4.String(config.opts.text)),
+		}
+	} else if config.opts.number != 0 {
+		value = githubv4.ProjectV2FieldValue{
+			Number: githubv4.NewFloat(githubv4.Float(config.opts.number)),
+		}
+	} else if config.opts.date != "" {
+		value = githubv4.ProjectV2FieldValue{
+			Date: githubv4.NewDate(githubv4.Date{Time: date}),
+		}
+	} else if config.opts.singleSelectOptionID != "" {
+		value = githubv4.ProjectV2FieldValue{
+			SingleSelectOptionID: githubv4.NewString(githubv4.String(config.opts.singleSelectOptionID)),
+		}
+	} else if config.opts.iterationID != "" {
+		value = githubv4.ProjectV2FieldValue{
+			IterationID: githubv4.NewString(githubv4.String(config.opts.iterationID)),
+		}
+	}
+
+	return &UpdateProjectV2FieldValue{}, map[string]interface{}{
+		"input": githubv4.UpdateProjectV2ItemFieldValueInput{
+			ProjectID: githubv4.ID(config.opts.projectID),
+			ItemID:    githubv4.ID(config.opts.itemID),
+			FieldID:   githubv4.ID(config.opts.fieldID),
+			Value:     value,
+		},
+	}
+}
+
+func printDraftIssueResults(config editItemConfig, item queries.DraftIssue) error {
+	// using table printer here for consistency in case it ends up being needed in the future
+	config.tp.AddField("Title")
+	config.tp.AddField("Body")
+	config.tp.EndRow()
+	config.tp.AddField(item.Title)
+	config.tp.AddField(item.Body)
+	config.tp.EndRow()
+	return config.tp.Render()
+}
+
+func printDraftIssueJSON(config editItemConfig, item queries.DraftIssue) error {
+	b, err := format.JSONProjectDraftIssue(item)
+	if err != nil {
+		return err
+	}
+	config.tp.AddField(string(b))
+	return config.tp.Render()
+}
+
+func printItemResults(config editItemConfig, item *queries.ProjectItem) error {
+	config.tp.AddField("Type")
+	config.tp.AddField("Title")
+	config.tp.AddField("Number")
+	config.tp.AddField("Repository")
+	config.tp.AddField("ID")
+	config.tp.EndRow()
+
+	config.tp.AddField(item.Type())
+	config.tp.AddField(item.Title())
+	if item.Number() == 0 {
+		config.tp.AddField(" - ")
+	} else {
+		config.tp.AddField(fmt.Sprintf("%d", item.Number()))
+	}
+	if item.Repo() == "" {
+		config.tp.AddField(" - ")
```

Comment:
Is this a placeholder for values that are missing? Could we consider just leaving it empty?

Reply:
when printing rows having an empty value seemed clearer to me, particularly across columns. it can definitely be changed if you don't agree!